examples/texture-arrays/main.rs

```diff
@@ -131,56 +132,6 @@ impl framework::Example for Example {
         let index_buffer = device
             .create_buffer_with_data(bytemuck::cast_slice(&index_data), wgpu::BufferUsage::INDEX);
 
-        let uniform_workaround_data = if uniform_workaround {
```

Comment:
holy smokes, the whole workaround is removed?

Reply:
The workaround is still there, but instead of a bunch of buffer stuff, I just need to push a push constant and be done, it's beautiful.